src/cesium_ion.rb

```diff
@@ -0,0 +1,32 @@
+#-------------------------------------------------------------------------------
```

Comment:
Is this header used for anything?  If not I would just remove it (if it is, then we should fill it in)